src/components/Menu/index.js

```diff
@@ -418,6 +418,16 @@ const SbMenuButton = {
     },
 
     isBorderless: Boolean,
+
+    hasIconRight: {
```

Comment:
Also, I think we should have something for icons on the left because currently we don't have such thing. I believe it would be useful. Usage case:

<img width="295" alt="Captura de Tela 2022-06-27 às 16 06 05" src="https://user-images.githubusercontent.com/26799272/176016805-adb95147-7972-4ee9-8f80-088b80fa08a8.png">

Reply:
Hi @fariacc, thanks for your comments!
First item: It's a good idea, I'll check here, and change this prop to "iconRight", because we have an "icon" prop and it can be confusing if we put "iconName".
Second comment: We can now do this implementation with the props we already have, I created a new story for you to see.
http://URL/?path=/story/design-system-components-sbmenu--with-icon-left